.pre-commit-config.yaml

```diff
@@ -1,4 +1,4 @@
-exclude: dist/.*
+exclude: ^(dist/.*|gradlew)$
```

Comment:
This file only affects the files in this repo. I think there isn't a  gradlew file in it. Are you trying to ignore it from being modified in a specific repository? I think you would have to add the exclusion there.

Reply:
My hope was that these exclusions would propagate to all repos that use this GHA.